src/components/EventList.js

```diff
@@ -35,14 +36,19 @@ export const EventList = ({
     }
   };
 
-  const renderItem = useRenderItem(QUERY_TYPES.EVENT_RECORDS, navigation, { noSubtitle });
+  const renderItem = useRenderItem(QUERY_TYPES.EVENT_RECORDS, navigation, {
+    noSubtitle
+  });
 
   const sectionedData = data.reduce((previous, current) => {
     if (previous[previous?.length - 1]?.title === current.params.details.listDate) {
       previous[previous.length - 1].data.push(current);
       return previous;
     } else {
-      previous.push({ title: current.params.details.listDate, data: [current] });
+      previous.push({
+        title: current.params.details.listDate,
+        data: [current]
+      });
```

Comment:
this was linted by you as well because of some eslint problems occurred within your development environment?

Reply:
I only got ESlint errors over punctuation marks.

Reply:
maybe some linting issue, where the max line length is setup to 80 at your machine? we use 100.
but no problem, i will check this before merge and correct if necessary. same in EventList.js.

Reply:
we will keep the new one for better readability of the object